### DISCUSSION TITLE
1.19.0 + Wear OS 2.9.0

### DISCUSSION THREAD

**Discussion by jamie-mh** [stratumauth/app#754] [General]
- **Visual**: Material You design with dynamic colour
- **New feature**: Wear OS tile (replaces favourite system)
- **New feature**: Import from FreeOTP
- **New feature**: Import from LastPass Authenticator
- **General**: Stronger backup encryption
- **General**: 3 new icons
- **General**: Translation updates

<hr /><em>This discussion was created from the release <a href='https://github.com/jamie-mh/AuthenticatorPro/releases/tag/1.19.0'>1.19.0 + Wear OS 2.9.0</a>.</em>

**Comment by zoroya1** [comment]
Material you theme looks nice 🔥😍 thanks for this 😄❤️
